src/server/pachyderm_bench_test.go

```diff
@@ -75,7 +75,6 @@ func benchmarkFiles(b *testing.B, fileNum int, minSize uint64, maxSize uint64, l
 	commit, err := c.StartCommit(repo, "master")
 	require.NoError(b, err)
 	if !b.Run(fmt.Sprintf("Put%dFiles", fileNum), func(b *testing.B) {
-		b.N = 1
```

Comment:
I have no idea what this is, but are you sure it doesn't have side-effects?

Reply:
So when you do benchmarks you're supposed to do them `b.N` times, which go changes for ops that are too short to measure reliably with a single run. Setting `b.N` is afaik completely meaningless, it's meant as a read only value. The benchmarks still aren't correct since they're not doing things `b.N` times. I skipped over that when I was changing things since there was a lot to change and this fixed the check and didn't make things less correct, I can add them in now.